rfcs/rfc-incorporation-application-staff-filing.md

```diff
@@ -0,0 +1,31 @@
+- Start Date: 2022-05-05
+- Target Major Version: 
+- Entity Issue: [9248](https://github.com/bcgov/entity/issues/9248)
+- Implementation PR: (leave this empty)
+
+# Summary
+
+This RFC outlines how the changes required for the staff to file an incorporation application. The UI flow
+from the staff dashboard (how to launch the Incorporation Application/Registration) is not in the scope of
+this RFC.
+
+# Detailed design
+
+For a registration/incorporation, the temporary registration is created using an account and the temp registration
```

Comment:
I suggest you clarify that this is the existing functionality for a regular user.